src/App.jsx

```diff
@@ -14,57 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const makeChoice = good => {
+    setSelectedGood(someGood => (someGood === good ? '' : good));
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        <>
+          {selectedGood ? `${selectedGood} is selected` : `No goods selected`}
+          {selectedGood && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => setSelectedGood('')}
```

Comment:
create `handleReset` function for it.
Use the `handle function` for `event handlers`.
Ur code will be more readable.
Also in the future - u will learn to optimize ur code (the number of component renders) using memoization.
```suggestion
              onClick={handleReset}
```

Reply:
what exactly do you mean?

Reply:
```
const handleReset = () => {
  setSelectedGood('');
}
```